app/src/check.js

```diff
@@ -209,6 +203,7 @@ export default styled(Check)`
   }
   .instruction {
     margin: 0 30px 20px;
+    font-size: 17px;
```

Comment:
We have set font sizes and try to avoid to add more variations than needed. Check out `app/catalog/branding/typography.md`